FormidableLabs/react-native-app-auth
# example project on rn 62.2

## Description
Bumps the example project up to rn version 62.2
Tested on iOS and Android 👍 
## Steps to verify
Start an emulator and check out that flippin' network activity!!
![flipper_network_reqs](https://user-images.githubusercontent.com/13589602/79383840-ba517800-7f1a-11ea-8b56-a2b42fdc3d6a.gif)
